config/leviton/vrcs4.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Product Revision="1" xmlns="https://github.com/OpenZWave/open-zwave">
+  <MetaData>
+    <MetaDataItem name="Name">VRCS4</MetaDataItem>
+    <MetaDataItem name="Description">4-Button Scene Controller with or without Switches</MetaDataItem>
+    <MetaDataItem name="OzwInfoPage">http://www.openzwave.com/device-database/001D:0261:0702</MetaDataItem>
+    <MetaDataItem name="ProductPic">images/leviton/vrcs4.png</MetaDataItem>
+    <MetaDataItem id="0261" name="ZWProductPage" type="0802">https://products.z-wavealliance.org/products/318/</MetaDataItem>
+    <MetaDataItem id="0261" name="FrequencyName" type="0802">U.S. / Canada / Mexico</MetaDataItem>
+    <MetaDataItem id="0261" name="Identifier" type="0802">VRCS4-M0Z</MetaDataItem>
+    <MetaDataItem id="0243" name="ZWProductPage" type="1302">https://products.z-wavealliance.org/products/287/</MetaDataItem>
+    <MetaDataItem id="0243" name="FrequencyName" type="1302">U.S. / Canada / Mexico</MetaDataItem>
+    <MetaDataItem id="0243" name="Identifier" type="1302">VRCS4-MRZ</MetaDataItem>
+    <ChangeLog>
+      <Entry author="Jeff Brown - jeff.brown@gmail.com" date="29 June 2020" revision="1">Initial creation</Entry>
+    </ChangeLog>
+  </MetaData>
+
+  <!-- Association Groups -->
+  <CommandClass id="133">
+    <Associations num_groups="4">
+      <!-- Each button toggles an individual scene on or off.  Only one scene can be active at once.
+           If the device has a switch, then by default it will associate group 1 with the local load.
+           Do not auto associate the controller with group 1. -->
+      <Group index="1" max_associations="232" label="Button 1" auto="false"/>
```

Comment:
same comment as above.

Reply:
Similarly, the VRCS4 presents its load as a separate device and auto-associated it to button 1 as the factory default.  That said, 3 of the 4 buttons are useless out of the box unless additional associations are made.

So we could argue that auto-association with the primary controller is more essential for this one.  I've made those associations manually for my case along with other configurations documented in this file.

Unfortunately, openzwave currently doesn't know how to configure the scenes such that it can distinguish between each button and it doesn't handle incoming scene actuator commands either.  With that in mind, I don't think auto-association is a good idea just yet but I will defer to your judgement on the matter.